app/views/gardens/index.html.haml

```diff
@@ -22,43 +21,38 @@
 
   %table.table.table-striped
     %tr
+      %th 
       - unless @owner
         %th Owner
       %th Garden name
-      %th Description
-      %th Location
       %th Area
-      %th Active?
+      %th Location
       %th Plantings
-      %th
 
     - @gardens.each do |garden|
       %tr
+        
+        %td.status
```

Comment:
Oh, I see what you mean. The trouble is that all this CSS is in a global namespace so it's pretty easy to create conflicts with generic names like "status" or "area".

Can we change "status" to "garden-status" and remove the "name" and "area" classes?